src/Magnum/Trade/PhongMaterialData.h

```diff
@@ -124,6 +123,48 @@ class MAGNUM_TRADE_EXPORT PhongMaterialData: public AbstractMaterialData {
          */
```

Comment:
Here I would mention that

```suggestion
         *
         * All `*CoordinateSet`s are implicitly set to @cpp 0 @ce.
         */
```